Cargo.toml

```diff
@@ -26,9 +26,10 @@ openssl = "0.10.55"
 serde = { version = "1.0.127", features = ["derive"] }
 serde_yaml = "0.9.13"
 sha2 = "0.10.2"
-tokio = { version = "1.24.2", features = ["rt", "process", "net", "io-util", "macros", "signal", "time"] }
+tokio = { version = "1.24.2", features = ["full"] }
 tokio-openssl = "0.6.2"
 tokio-stream = { version = "0.1.7", features = ["io-util"] }
+multer = { version="2.1.0", features = ["tokio-io"] }
```

Comment:
This should be in the `dev-dependencies` since it's only used for the example :)